src/components/Title/Title.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Title extends React.Component {
```

Comment:
As this doesn't use state at all (yet, anyway) we can use a stateless component https://reactjs.org/docs/components-and-props.html#function-and-class-components

Reply:
On it